tasks/Resources/Provisioning/Hive/GCP/gcp-cluster.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: v1
+items:
+- apiVersion: hive.openshift.io/v1
+  kind: ClusterPool
+  metadata:
+    name: {{ hive_cluster_name|default("rhodsgcp") }}
+    namespace: rhods
+  spec:
+    baseDomain: gcp.rhods.ccitredhat.com
+    imageSetRef:
+      name: {{ image_set|default("rhods-openshift") }}
+    platform:
+      gcp:
+        region: {{ gcp_region|default("us-central1") }}
+        credentialsSecretRef:
+          name: gcp-creds
+    installConfigSecretTemplateRef:
+      name: gcp-sno-install-config
+    size: 1
+    skipMachinePools: true
+- apiVersion: v1
+  kind: Secret
+  stringData:
+    install-config.yaml: |
+      apiVersion: v1
+      baseDomain: rhods.ccitredhat.com
+      compute:
+      - name: worker
+        platform:
+          gcp:
+            type: {{ worker_node_instance_type|default("custom-8-32768") }}
+        replicas: {{ worker_node_replicas|default("2") }}
+      controlPlane:
+        name: master
+        platform:
+          gcp:
+            type: custom-8-32768
```

Comment:
Could we expose master nodes platform type to get inputs from user and defaults to this value?

Reply:
changed